rainier-core/src/main/scala/com/stripe/rainier/core/SBCModel.scala

```diff
@@ -0,0 +1,48 @@
+package com.stripe.rainier.core
+
+import com.stripe.rainier.compute._
+import com.stripe.rainier.sampler._
+
+trait SBCModel[L, T] {
```

Comment:
could drop the type params if we moved to goldset always being Double

Reply:
Still working on this.

Reply:
Done